src/cfc/sync.ts

```diff
@@ -10,23 +10,23 @@ export default (flicking: Flicking, diffResult: DiffResult<any>, rendered: any[]
     let startIdx = -1;
     let prevIdx = -1;
 
-    diffResult.removed.forEach((removedIdx, idx) => {
+    [...diffResult.removed].reverse().forEach(removedIdx => {
```

Comment:
If the order is correct, the index will change every time you delete it, is that ok?

Reply:
Good point, I should add a TC for that